neptune-export/src/main/java/com/amazonaws/services/neptune/ExportPropertyGraphFromGremlinQueries.java

```diff
@@ -94,7 +98,7 @@ public void run() {
                             directories.queriesResource();
 
                     CsvPrinterOptions csvPrinterOptions = CsvPrinterOptions.builder().setIncludeTypeDefinitions(includeTypeDefinitions).build();
-                    JsonPrinterOptions jsonPrinterOptions = JsonPrinterOptions.builder().setStrictCardinality(true).build();
+                    JsonPrinterOptions jsonPrinterOptions = JsonPrinterOptions.builder().setStrictCardinality(strictCardinality).build();
```

Comment:
This is a change to the default behavior of `setStrictCardinality(true)`.  Instead of changing the default behavior, can we leave `strictCardinality` as `true` and change the parameter to accept `--strict-cardinality false` instead?